apps/explorer/src/pages/id/utils.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { useGetTransaction } from '~/hooks/useGetTransaction';
+import { trimStdLibPrefix } from '~/utils/stringUtils';
+import { getOwnerStr } from '~/utils/objectUtils';
+import { type DataType } from '~/pages/object-result/ObjectResultType';
+
+export const GENESIS_TX_DIGEST = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=';
+
+export function usePackageViewedData({ data }: { data: DataType }) {
```

Comment:
This hook seems like it's some weird view model, I don't know why we need this, doesn't seem useful.